navigation/AppNavigator.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { createStackNavigator } from '@react-navigation/stack';
+import { Platform } from 'react-native';
+import { Ionicons } from '@expo/vector-icons';
+
+import Colors from '../theme/Colors';
+
+import DashboardScreen from '../screens/DashboardScreen';
+import SalesScreen, { SalesScreenNavOptions } from '../screens/SalesScreen';
+import SalesEntryScreen from '../screens/SalesEntryScreen';
+import CostsScreen, { CostsScreenNavOptions } from '../screens/CostsScreen';
+import CostsEntryScreen from '../screens/CostsEntryScreen';
+import SettingsScreen from '../screens/SettingsScreen';
+
+const defaultNavigationOptions = {
```

Comment:
Type needed.

Reply:
It should stay here because that options are just for AppNavigator or we could move it to separate module but in this case we should create new folder for AppNavigator and maybe another one for AuthNavigator - AuthNavigator could had in future also own navigation options. What you think ?

Reply:
Regarding type - it seems typing react navigation is a little bit more complicated as I expected. I found example 
https://github.com/react-navigation/stack/blob/master/src/types.tsx 
but I have no idea how to deal with it. Help needed

Reply:
In my opinion, creating separate directories for each navigator seems to be good idea.
And regarding types - this thing `defaultNavigationOptions` should be of type DefaultNavigatorOptions (or something like that, I can't check now).

Reply:
Solved.